src/uct/tcp/sockcm/sockcm_md.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_SOCKCM_MD_H_
+#define UCT_SOCKCM_MD_H_
+
+#include "sockcm_def.h"
+#include <uct/base/uct_md.h>
+#include <ucs/sys/sock.h>
+#include <ucs/time/time.h>
+
+/**
+ * SOCKCM memory domain.
+ */
+typedef struct uct_sockcm_md {
+    uct_md_t                 super;
+    double                   addr_resolve_timeout; // FIXME
```

Comment:
here and below C++ comments in C code. pls, change to `/**/`

Reply:
fixed

Reply:
not fixed..